Brokerages/Fxcm/FxcmBrokerage.DataQueueHandler.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Linq;
+using com.fxcm.fix;
+using com.fxcm.fix.pretrade;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Logging;
+using QuantConnect.Packets;
+
+namespace QuantConnect.Brokerages.Fxcm
+{
+    public partial class FxcmBrokerage
+    {
+        private readonly List<Tick> _ticks = new List<Tick>();
+        private readonly HashSet<string> _subscribedSymbols = new HashSet<string>(); 
```

Comment:
However, it looks like the `IDataQueueHandler.Subscribe` function needs to be updated to transport `Symbol` instances instead of strings, so for now, let's just modify this to be a `HashSet<Symbol>` and we'll add the signature change to the list.